api/app/signals/apps/services/tests/domain/test_pdf_summary.py

```diff
@@ -158,6 +158,26 @@ def test_show_contact_details(self):
         self.assertIn('foo@bar.com', html)
         self.assertIn('0612345678', html)
 
+    @override_settings(FEATURE_FLAGS={'SHOW_REPORTER_CONTACT_DETAILS_IN_PDF': False})
+    def test_no_contact_details_when_feature_flag_is_disabled(self):
+        """
+        When the feature flag SHOW_REPORTER_CONTACT_DETAILS_IN_PDF is disabled
+        the PDF export should not contain any personal details
+        """
+
+        user = UserFactory.create()
+        sia_can_view_contact_details = Permission.objects.get(codename='sia_can_view_contact_details')
+        user.user_permissions.add(sia_can_view_contact_details)
+        user = User.objects.get(pk=user.id)
+
+        self.assertTrue(user.has_perm('signals.sia_can_view_contact_details'))
+
+        # Check if the HTML does not contain the contact details
+        # although the signal does contain these details
+        html = PDFSummaryService._get_html(self.signal, None, True)
+        self.assertNotIn('foo@bar.com', html)
```

Comment:
These are set implicitly in the factory, but make them explicit by setting them in the call to `UserFactory.create()`.

Reply:
Resolved this